pgmax/fg/nodes.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import annotations
```

Comment:
I wouldn't drop these (actually, I'd always add them) - there are all sorts of annoyances that can crop up with ordering of modules/functions inside files and type annotations.

Reply:
If mypy is not complaining then probably fine to get rid of some of these? It's only useful for some of the modules and with mypy check enforced we don't need to worry about missing failure cases